sound/soc/bcm/snd-soc-tda1543.c

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
The file looks unchanged to me, and still has the wrong name (tda1543.c would be better).